src/bb8_backup.py

```diff
@@ -0,0 +1,43 @@
+from subprocess import run
+
+targets = ["orderly"]
+finished_setup = False
+bb8_dir = "../montagu-bb8/bb8"
+
+
+def requires_bb8_setup(func):
+    def decorated(*args, **kwargs):
+        global finished_setup
+        if not finished_setup:
+            setup()
+        return func(*args, **kwargs)
+
+    return decorated
+
+
+def setup():
+    global finished_setup
+    template = "- Configuring and installing bb8 backup service with these " \
+               "targets: {}"
+    print(template.format(targets))
+    args = ["./setup.sh", "../config.json"] + targets
+    run(args, cwd=bb8_dir, check=True)
+    finished_setup = True
+
+
+@requires_bb8_setup
+def backup():
+    print("Performing bb8 backup")
+    run(["bb8", "backup"], check=True)
```

Comment:
Out of scope for this ticket but if we provided bb8 as a python module we'd be able to centralise a lot of the logic in this file for use on subsequent projects?  If that sounds useful I can open a ticket

Reply:
That sounds good :)

Reply:
https://vimc.myjetbrains.com/youtrack/issue/VIMC-1699